test/arbitrary/ArbitaryPartyDetails.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as jsc from 'jsverify';
+import {PartyDetails} from "../../src/model/PartyDetails";
+
+export const ArbitraryPartyDetails: jsc.Arbitrary<PartyDetails> = jsc
```

Comment:
Despite most of the classes still using `tuple`, moving forward classes should use the more apt `record` method. Have a look at `ArbitraryGetActiveContractsResponse` to get a sense of how it works.

Reply:
done